src/runtime/getPositions.test.ts

```diff
@@ -151,4 +153,90 @@ describe(getPositions, () => {
       "Positions hook for app 'test-hook' does not implement 'getAppTokenDefinition'. Please implement it to resolve the intermediary app token definition for 0x1e593f1fe7b61c53874b54ec0c59fd0d5eb8621e (celo-mainnet)",
     )
   })
+
+  it("should use the fallbackPriceUsd if the token can't be found", async () => {
+    const testHook: PositionsHook = {
+      getInfo() {
+        return {
+          id: 'beefy-price-escape',
+          name: 'Beefy Price Escape',
+          description: 'Beefy with price escape hatch for underlying tokens',
+        }
+      },
+
+      async getPositionDefinitions(networkId: NetworkId, _address: string) {
+        if (networkId !== NetworkId['op-mainnet']) {
+          return []
+        }
+
+        return [
+          {
+            type: 'app-token-definition',
+            networkId,
+            // Beefy crvUSD/WBTC/WETH Vault
+            // https://app.beefy.com/vault/curve-op-tricrypto-crvusd
+            address: '0xf82160Bad52C235102174aE5E7f36d5099DEEad3',
+            tokens: [
+              {
+                // Underlying Curve LP token
+                address: '0x4456d13Fc6736e8e8330394c0C622103E06ea419',
+                networkId,
+                // Fallback until we can properly decompose the token into base tokens
+                fallbackPriceUsd: '0.5' as SerializedDecimalNumber,
+              },
+            ],
+            // Meaning that 1 share of the vault is worth 2 underlying token
+            pricePerShare: [toDecimalNumber(2n, 0)],
```

Comment:
@gnardini this should interest you for implementing beefy vaults as `app-token-definition`